src/Manager.ts

```diff
@@ -130,7 +130,7 @@ export class Manager extends EventEmitter {
    * @since 1.0.0
    */
   public async stateUpdate(update: VoiceState): Promise<void> {
-    const player = this.players.get(update.guild_id);
+    const player = this.players.get(update.guild_id ?? "");
```

Comment:
This can be better-handled by adding a check before retrieval of value from the Map.
```ts
if (!update.guild_id) return;
```